src/pages/EditorPage/EditorPanel.jsx

```diff
@@ -644,36 +677,60 @@ const EditorPanel = ({
                     />
                   )
                 } else if (attrib?.enum) {
+                  const getInheritLabel = (attrib, parentValue, isMultiSelect) => {
+                    if (!parentValue) {
+                      return
+                    }
+                    if (!isMultiSelect) {
+                      return (
+                        'inherited - ' + attrib.enum.filter((el) => el.value === parentValue)[0]?.label || null
+                      )
+                    }
+                    const labels = attrib.enum
+                      ?.filter((el) => parentValue.includes(el.value))
+                      .map((el) => el.label || el.value)
+                    return `inhreited - (${labels.join(', ')})`
+                  }
+
                   // dropdown
                   const isMultiSelect = ['list_of_strings'].includes(attrib?.type)
                   let enumValue = isMultiSelect ? value : [value]
                   if (multipleValues) {
                     enumValue = isMultiSelect ? union(...multipleValues) : multipleValues
                   }
 
+                  let options = attrib.enum
+                  const inheritedOptionLabel = getInheritLabel(attrib, parentValue, isMultiSelect)
+                  if (inheritedOptionLabel) {
+                    placeholder = inheritedOptionLabel
+                    options = [{ value: null, label: inheritedOptionLabel }, ...attrib.enum]
+                  }
+
                   // never show value when inherited, just show placeholder
                   if (!isOwn) enumValue = null
-
                   input = (
                     <Dropdown
```

Comment:
Overflow styling issue. Maybe it's an issue ARC?

![image](https://github.com/user-attachments/assets/71ce7efa-be96-4e00-8af6-5070a7251c41)